src/js/hero.js

```diff
@@ -0,0 +1,213 @@
+'use strict';
+const refst = {
+  heroList: document.querySelector('.hero-list'),
+  heroTitle: document.querySelector('.hero-text'),
+  hero: document.querySelector('.hero'),
+  select: document.querySelector('.hero-select'),
+
+
+  isHiden: document.querySelector('.is-hiden'),
+  coctailTitel: document.querySelector('.coctails-section__title'),
+  cocktalisTitel: document.querySelector('.coctails-section__coctails-list'),
+ 
+
+  herospan : document.querySelector('.hero-span'),
+  heroBox : document.querySelector('.hero-div'),
+  heroSelect : document.querySelector('.hero-boxList'),
+  heroListUl:document.querySelector('.hero-list__ul'),
+ 
+  headerinput:document.querySelector('.header-input'),
+};
+
+
+// import mainFunction from './coctails';
+const coctailsList = document.querySelector('.coctails-section__coctails-list');
+
+let arrayLength = 0;
+
+const { heroList, heroTitle, hero, select, isHiden, 
+  heroItem ,headerinput,herospan,coctailTitel,cocktalisTitel,heroBox,
+  heroSelect,heroListUl } = refst;
+
+const heroTitleImg = () => {
+  return `<div class="hero-container">
+</div>`;
+};
+
+function creaitMarkapArr(e) {
```

Comment:
TODO: rename the function (mb You mean createMarkupArr) and change in later code